deepinv/unfolded/unfolded.py

```diff
@@ -61,36 +60,41 @@ def __init__(
             self.iterator.f_step = custom_f_step
 
 
-def Unfolded(
-    algo,
-    trainable_params=[],
-    data_fidelity=L2(),
-    F_fn=None,
-    g_first=False,
-    beta=1.0,
-    bregman_potential="L2",
-    **kwargs
+def unfolded_builder(
+    iteration, data_fidelity=L2(), F_fn=None, g_first=False, beta=1.0, **kwargs
 ):
     r"""
     Function building the appropriate Unfolded architecture.
 
     :param algo: either name of the algorithm to be used, or an iterator. If an algorithm name (string), should be either `"PGD"`, `"ADMM"`, `"HQS"`, `"CP"` or `"DRS"`.
-    :param deepinv.optim.DataFidelity data_fidelity: data fidelity term in the optimization problem.
-    :param F_fn: Custom user input cost function. Default: None.
-    :param g_first: whether to perform the step on :math:`g` before that on :math:`f` before or not. Default: False.
+    :param deepinv.optim.data_fidelity data_fidelity: data fidelity term in the optimisation problem.
```

Comment:
This change is not correct
- The class if DataFidelity
- We use american english